### PR TITLE
ping: add Indonesian translation

### PR DESCRIPTION
**Pull request by apraditya**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.id/common/ping.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# ping
+
+> Mengirim paket-paket ICMP ECHO_REQUEST ke host dalam jaringan.
+> Informasi lebih lanjut: <https://manned.org/ping>.
+
+- Ping _host_:
```
Comment (kbdharun) [inline pages.id/common/ping.md:6]:
```suggestion
- Ping host:
```

File: pages.id/common/ping.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`ping {{host}}`
+
+- Ping _host_ dalam jumlah kali tertentu:
```
Comment (kbdharun) [inline pages.id/common/ping.md:10]:
```suggestion
- Ping host dalam jumlah kali tertentu:
```

File: pages.id/common/ping.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`ping -c {{jumlah}} {{host}}`
+
+- Ping _host_, dengan menentukan interval dalam sekian detik di antara request (asalnya 1 detik):
```
Comment (kbdharun) [inline pages.id/common/ping.md:14]:
```suggestion
- Ping host, dengan menentukan interval dalam sekian detik di antara request (asalnya 1 detik):
```

File: pages.id/common/ping.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`ping -i {{jumlah-detik}} {{host}}`
+
+- Ping _host_ tanpa mencoba untuk melihat alamat dari nama-nama simbolis:
```
Comment (kbdharun) [inline pages.id/common/ping.md:18]:
```suggestion
- Ping host tanpa mencoba untuk melihat alamat dari nama-nama simbolis:
```

File: pages.id/common/ping.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`ping -n {{host}}`
+
+- Ping _host_ dan membunyikan bel saat paket diterima (jika terminal anda mendukungnya):
```
Comment (kbdharun) [inline pages.id/common/ping.md:22]:
```suggestion
- Ping host dan membunyikan bel saat paket diterima (jika terminal anda mendukungnya):
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page. 

We don't use `_` in our descriptions.

**Review by kbdharun — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**